src/l3.c

```diff
@@ -242,6 +271,28 @@ l3_init(const char *path)
     return 0;
 }
 
+/**
+ * ****************************************************************************
+ * Initialize L3's logging sub-system to use fprintf() to named `path`.
+ */
+int
+l3_init_fprintf(const char *path)
+{
+    if (!path) {
+        fprintf(stderr, "%s: Invalid arg 'path'=%p\n", __func__, path);
+        return -1;
+    }
+
+    l3_log_fh = fopen(path, "w+");
+    if (!l3_log_fh) {
+        fprintf(stderr, "%s: Error opening log-file at '%p'. Error=%d\n",
+                __func__, path, errno);
```

Comment:
Minor: you could use `strerror()` to give a more meaningful error message. Or come to that, `perror()`

Reply:
Sounded like a good idea, so I added this.

Upon unit-testing, I discovered that the existing error raising mechanism in this core programs are already reporting what seems to be like an `perror` output:

See this output, when I changed an output file to `const char *logfile = "/xxtmp/l3.c-server-test.dat";`

```
agurajada-Linux-Vm:[939] $ build/release/bin/use-cases/svmsg_file_server
l3_init_fprintf: Error opening log-file at '0x5de2ee48760b'. Error=2
fopen failed: No such file or directory
ERROR [ENOENT No such file or directory] l3_log_init
```

So, I think a specific addition of `perror()` in this one place is not needed.

While investigating this, it occurred to me that we should enhance this errExit() and similar interfaces to integrate with LOC. I have opened a new dev-issue #68 to track this potential enhancement.